app/src/userpages/components/StreamPage/Show/InfoView/index.jsx

```diff
@@ -97,28 +98,24 @@ export const InfoView = ({ disabled }: Props) => {
                     autoComplete="off"
                 />
             </div>
-            {stream && stream.id &&
-                <React.Fragment>
-                    <div className={styles.textInput}>
-                        <TextInput
-                            label={I18n.t('userpages.streams.edit.details.streamId')}
-                            type="text"
-                            name="id"
-                            value={(stream && stream.id) || ''}
-                            preserveLabelSpace
-                            readOnly
-                            disabled={disabled}
-                            actions={[
-                                <DropdownActions.Item key="copy" onClick={() => onCopy(stream.id)}>
-                                    <Translate value="userpages.keyField.copy" />
-                                </DropdownActions.Item>,
-                            ]}
-                        />
-                    </div>
-                    <h5 className={styles.partitions}>{I18n.t('userpages.streams.edit.details.partitions')}</h5>
-                    <PartitionsView disabled={disabled} />
-                </React.Fragment>
-            }
+            <div className={styles.textInput}>
+                <TextInput
+                    label={I18n.t('userpages.streams.edit.details.streamId')}
+                    type="text"
+                    name="id"
+                    value={(stream && stream.id) || ''}
+                    preserveLabelSpace
+                    readOnly
+                    disabled={disabled}
+                    actions={[
```

Comment:
This is an edge case - the text input should be disabled, but its actions should never be. I.e. If I have read-only permission access to a stream, I should still be able to `copy Stream ID`.